Engine/DataFeeds/ZipDataCacheProvider.cs

```diff
@@ -265,7 +302,15 @@ public CachedZipFile(ZipFile data, string key)
         {
             _data = data;
             _key = key;
-            _dateCached = DateTime.Now;
+            DateCached = DateTime.Now;
+        }
+
+        /// <summary>
+        /// Method used to update the last time this file was accessed.
+        /// </summary>
+        public void Refresh()
+        {
+            DateCached = DateTime.Now;
```

Comment:
Prefer usage of `DateTime.UtcNow` where possible.

To be true, `DateTime.Now` documentation does state that the system local time zone is cached upon application start, so if the system time zone changed it shouldn't affect `DateTime.Now` (unless `TimeZoneInfo.ClearCachedData()` is called). That being said, I still find it safer (and more consistent) to use UTC where possible.